Replaced *.quay.io allowlist wildcard with discrete list of subdomains (#41850)
openshift/openshift-docs: modules/configuring-firewall.adoc

|443, 80
|Provides core container images

|`cdn.quay.io`
|443, 80
|Provides core container images

|`cdn01.quay.io`
|443, 80
|Provides core container images

|`cdn02.quay.io`
|443, 80
|Provides core container images

|`cdn03.quay.io`
|443, 80
|Provides core container images

|`sso.redhat.com`
|443, 80
|The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`

|`rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com`
|443, 80
|Provides {op-system-first} images

|===
+

> You may also use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn0[1-3].quay.io` and `rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com` in your allowlist.

Reviewer: > ! may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.

```suggestion
You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn0[1-3].quay.io` and `rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com` in your allowlist.
```
